test/validation/api/time/time.c

```diff
@@ -436,12 +436,12 @@ static void time_test_accuracy(time_cb time_cur, time_from_ns_cb time_from_ns)
 		wait = odp_time_sum(wait, sec);
 	}
 
+	c2 = time(NULL);
 	t2 = time_cur();
-	c2 = clock();
 
 	diff  = odp_time_diff(t2, t1);
 	sec_t = ((double)odp_time_to_ns(diff)) / ODP_TIME_SEC_IN_NS;
-	sec_c = ((double)(c2 - c1)) / CLOCKS_PER_SEC;
+	sec_c = ((double)(c2 - c1));
```

Comment:
time_t is an integer value of seconds, right? sec_c is the reference that we compare ODP time measurement against. Test passes if ODP measures within +-5% the reference. Usage of time() change the reference to be very coarse grained (e.g. 4, 5 or 6 sec), while the real time spent on the wait loop may be e.g. 5.453 sec and ODP may measure 5.461 sec and the time() measures just 5.0 sec. So, test result would be "fail" (5.461 vs 5.0) although ODP measurement error was only+0.14% (5.461 vs 5.453).

Reply:
I will change that to `gettimeofday()`.